Dockerfile

```diff
@@ -23,6 +23,10 @@ RUN mkdir -p source_declarative_manifest \
 # Remove unnecessary build files
 RUN rm -rf dist/ pyproject.toml poetry.lock README.md
 
+# Set ownership of /airbyte to the airbyte user
```

Comment:
@alafanechere - Is this a correct point, and if so, wdyt of including here for clarity?

```suggestion
# Set ownership of /airbyte to the (non-root) airbyte user
```